omod/src/main/webapp/resources/scripts/patientsearch/patientSearchWidget.js

```diff
@@ -315,7 +315,8 @@ function PatientSearchWidget(configuration){
     var updateSearchResults = function(results){
         var dataRows = [];
         if(results){
-            var results = removeDuplicates(results);
+            //first remove any null objects from the results
```

Comment:
nit, at the end of the day, the line below removes null objects and then removes duplicates, correct?  (the way the comment is worded I read it to mean all the line does is remove null elements...
0

Reply:
Thanks @mogoodrich ! I have updated the comments.